templates/solaris-0.1.2/requirements.yml

```diff
@@ -0,0 +1,34 @@
+channels:
+  - pytorch
+  - conda-forge
+  - defaults
+dependencies:
```

Comment:
Do we need this section? What happens when a user changes these?

Reply:
Just like with requirements.txt for pytorch, it allows users to know what's installed and have some feedback. If those variables change then a new build op will happen and the old versions may get evicted. A comment here similar to our other dependency based package sets might be a good idea ("Don't change this unless you need to")